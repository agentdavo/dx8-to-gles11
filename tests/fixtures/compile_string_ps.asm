#define DIFF v0
#include "cs_ps/level1.inc"
ps.1.1
#include "cs_ps/ops.inc"
