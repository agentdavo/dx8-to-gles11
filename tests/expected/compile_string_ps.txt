TEX_SAMPLE stage=0
TEX_ENV_COMBINE func=8448
